client/app/components/ApplicationArea/ApplicationLayout/DesktopNavbar.jsx

```diff
@@ -113,6 +113,7 @@ export default function DesktopNavbar() {
             key="create"
             popupClassName="desktop-navbar-submenu"
             data-test="CreateButton"
+            tabIndex={0}
```

Comment:
When I tested, this element appears to have a `tabIndex`, but no keyboard accessibility (if I hit `enter` or `space`, nothing happens). It's also weird if you remove the `tabIndex` -- maybe create a ticket tracking the issue? I think it may be out of scope for this first pass, but we should definitely address it.

Reply:
This is actually caused by Ant. The only way to trigger interactivity here is by triggering `onMouseOver` event. Maybe we can do it programmatically in a future PR. WDYT?

Reply:
As long as there's a ticket for it, I'm fine with it.